po/ie.po

```diff
@@ -0,0 +1,6155 @@
+# Interlingue translation of geany
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# Olga Smirnova <mistresssilvara@hotmail.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-08-10 21:37+0700\n"
+"PO-Revision-Date: 2020-08-11 10:29+0700\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.12\n"
+"Last-Translator: OIS <mistresssilvara@hotmail.com>\n"
+"Language: ie\n"
+
+#: ../geany.desktop.in.h:1 ../data/geany.glade.h:343
+msgid "Geany"
+msgstr "Geany"
+
+#: ../geany.desktop.in.h:2
+msgid "Integrated Development Environment"
+msgstr "Integrat ambiente de developation"
+
+#: ../geany.desktop.in.h:3
+msgid "A fast and lightweight IDE using GTK+"
+msgstr "Un rapid e minimalistic IDE que usa GTK+"
+
+#: ../geany.desktop.in.h:4
+msgid "Text;Editor;"
+msgstr "textu;redactor;"
+
+#: ../data/geany.glade.h:1
+msgid "_Toolbar Preferences"
+msgstr "Preferenties del ins_trumentarium"
+
+#: ../data/geany.glade.h:2
+#, fuzzy
+msgid "_Hide Toolbar"
+msgstr "Panel de instrumentarium"
+
+#: ../data/geany.glade.h:3
+msgid "_Edit"
+msgstr "_Redacter"
+
+#: ../data/geany.glade.h:4
+msgid "_Format"
+msgstr "_Formate"
+
+#: ../data/geany.glade.h:5
+msgid "I_nsert"
+msgstr "I_nserter"
+
+#: ../data/geany.glade.h:6
+#, fuzzy
+msgid "Insert _ChangeLog Entry"
+msgstr "Diarium de modification"
+
+#: ../data/geany.glade.h:7
+msgid "Insert _Function Description"
+msgstr "Inserter descrition del _function"
+
+#: ../data/geany.glade.h:8
+msgid "Insert Mu_ltiline Comment"
+msgstr "Inserter mu_ltilineari commenta"
+
+#: ../data/geany.glade.h:9
+msgid "_More"
+msgstr "_Plu"
+
+#: ../data/geany.glade.h:10
+#, fuzzy
+msgid "Insert File _Header"
+msgstr "Inserter un _file"
+
+#: ../data/geany.glade.h:11
+msgid "Insert _GPL Notice"
+msgstr "Inserter anuncie _GPL"
+
+#: ../data/geany.glade.h:12
+msgid "Insert _BSD License Notice"
+msgstr "Inserter un licentie _BSD"
+
+#: ../data/geany.glade.h:13
+msgid "Insert Dat_e"
+msgstr "Ins_erter li date"
+
+#: ../data/geany.glade.h:14
+msgid "invisible"
+msgstr "ínvisibil"
+
+#: ../data/geany.glade.h:15
+msgid "_Insert \"include <...>\""
+msgstr "_Inserter «include <...>»"
+
+#: ../data/geany.glade.h:16 ../src/keybindings.c:513
+#, fuzzy
+msgid "Insert Alternative _White Space"
+msgstr "Inserter interspacie"
+
+#: ../data/geany.glade.h:17
+msgid "_Search"
+msgstr "_Serchar"
+
+#: ../data/geany.glade.h:18
+#, fuzzy
+msgid "Open Selected F_ile"
+msgstr "Crear un f_ile"
```

Comment:
Should this be "__Aperte__ li file __selecte__" or similar?

Reply:
Those are fuzzy (untranslated) strings.

Reply:
Yeah sorry I was just trying to help to defuzz by providing context. I know Geany well but not so much Interlingue :)